tests/integration/create_token.py

```diff
@@ -53,6 +53,19 @@ def create_token_without_jti(self, form_type_id, eq_id, **extra_payload):
 
         return self.generate_token(payload_vars)
 
+    def create_token_with_survey_url(self, form_type_id, eq_id, survey_url, **extra_payload):
```

Comment:
It might be worth creating a method that returns a payload to share between the other methods in this file, then adding the survey_url to that before calling generate_token.